middle_end/flambda/to_cmm/un_cps.ml

```diff
@@ -1253,15 +1254,18 @@ and let_dynamic_set_of_closures env res body closure_vars decls elts =
                  (List.map fst (Closure_id.Map.bindings decls))
                  (List.map fst (Var_within_closure.Map.bindings elts))
   in
-  let l, env, effs = fill_layout decls elts env Ece.pure [] 0 layout in
+  (* Allocating the closure has at least generative effects *)
+  let effs = Effects.Only_generative_effects Immutable,
+             Coeffects.No_coeffects in
```

Comment:
Style remark: I don't like this indentation style. If the `let` doesn't fit on one line, then everything after the `=` should be on a new line and the final `in` should be on its own line, in my opinion.

Reply:
Yeah, I agree.